### PR TITLE
feat: css overhaul

### PR DESCRIPTION
**Pull request by isaaceichelberger**
- `components/styles/*.css/` no longer used, all css moved to `index.css`
- removes duplicate classes
- all css for coloring now uses variables, so colors do not have to be looked up when adding new classes or making changes to classes - these are `formInSubForm` `subForm` `background` `red` `blue`, etc. All are visible at the top of `index.css`
- all components/forms now support dark mode (for fun). default is still light mode
- updated storybook to allow components to be visible in dark mode (click the sun/moon in the top of storybook)  - will require `npm i` in /storybook
image

### CODE REVIEW

**Review by jvallexm — APPROVED** [review]

File: src/components/UnderConstruction.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { WarningAlt } from "@carbon/icons-react";
  */
 const UnderConstruction = () => {
   return (
-    <div>
+    <div className="underConstruction">
```
Comment (evpirnia) [inline src/components/UnderConstruction.js:9]: can we add center alignment to the under construction pages?
Screen Shot 2023-03-17 at 11 36 38 AM

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+.cursorNotAllowed {
+  cursor: not-allowed;
+}
+
+.forceTertiaryButtonStyles {
+  padding-right: 0.4375rem !important;
+  padding-left: 0.4375rem !important;
+}
+.inlineBlock {
```
Comment (evpirnia) [inline src/index.css:234]: format please. should be lines b/w classes

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+  --formInSubForm: #fffdfd;
+  --cds-text-primary: #161616;
+  --cds-text-secondary: #525252;
+  --cds-field: #f4f4f4;
+  --cds-border-strong: #8d8d8d;
+  --blue: #0f62fe;
+  --red: #da1e28;
+}
+
+.dark {
```
Comment (evpirnia) [inline src/index.css:14]: description not readable in dark mode

Screen Shot 2023-03-17 at 11 46 42 AM
Screen Shot 2023-03-17 at 11 47 21 AM

**Review by evpirnia — CHANGES_REQUESTED** [review]

File: src/components/UnderConstruction.js (hunk 1 above)
Comment (isaaceichelberger) [reply src/components/UnderConstruction.js:9]: idk how this got messed up, but fixed

**Review by evpirnia — APPROVED** [review]
loveeee it, GJ ◡̈

**Pull request merged**